radixdlt/babylon-wallet-android
# ABW-2448 - Skip main seedphrase during backup restoration flow.

## Description
This PR add ability to skip importing main babylon seed phrase which was mandatory before during the backup restoration flow.

## How to test

1. Create some accounts on main net 
2. You can also import Olympia or create ledger accounts if you want
3. Export profile to the file / backup
4. Clear wallet data
5. Restore Wallet from backup and chose the file you exported profile to
6. On the Main Seed Phrase Required page, choose I don't have the main seed phrase button to skip importing it and continue
7. If you previously had Olympia accounts you can choose to either import its seedphrase or skip
8. Verify that babylon accounts you have skipped the seedphrase for are appearing on Dashboard but we don't have the seedphrase which you can verify in the Seed Phrases screen (see video and **Seed Phrase Entry Required** label)

Please take your time to test also existing functionality and different combination to see if nothing has been broken.

## Video
Here is happy path :
https://github.com/radixdlt/babylon-wallet-android/assets/108684750/144f6af1-d2c9-4795-8520-1ce38cf4d925

But you can also follow what Cyon showed on the video -> https://rdxworks.slack.com/archives/C031A0V1A1W/p1700051937477939

## PR submission checklist
- [x] I have tested backup restoration flow and verified that we can skip main seed phrase import
